dart/lib/src/sentry_trace_origins.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:meta/meta.dart';
+
+@internal
+class SentryTraceOrigins {
+  static const manual = 'manual';
+
+  static const autoNavigationSentryNavigatorObserver =
```

Comment:
I think we can focus the naming on the instrumented class instead of our class name, for example.
`auto.navigation.route_observer`, because the instrumented class here is `RouteObserver`.
`SentryNavigatorObserver` is just our class name, that can be renamed or something.

Reply:
@philipphofmann do you agree?

Reply:
See the point/question above, if this is the case we should also state so explicitly in the documentation.